prow/plugins/trigger/trigger.go

```diff
@@ -148,7 +148,7 @@ func isUserTrusted(ghc githubClient, user, trustedOrg, org string) (bool, error)
 	} else if orgMember {
 		return true, nil
 	}
-	if org != trustedOrg {
+	if trustedOrg != "" && org != trustedOrg {
```

Comment:
I this is already checked when we determine the trustedOrg before handling the event: https://github.com/kubernetes/test-infra/blob/ffc76a8fc5f0301a8d5e1be16d968ce376d63f5e/prow/plugins/trigger/trigger.go#L135-L142